### ISSUE TITLE
Cannot jump to or get autocompletion for packages installed by opam

### ISSUE THREAD

**Issue by 1yefuwang1** [reasonml-editor/vscode-reasonml#162]
I'm trying to use Reason to develop native programs instead of using OCaml. So I cloned [ReasonNativeProject](https://github.com/reasonml/ReasonNativeProject).  
I find that 'jump to source' and autocopletion work perfectly with the std library shipped with bucklescript:
![image](https://user-images.githubusercontent.com/18023393/35255608-ccae2244-002a-11e8-84d7-a1e45481453b.png)
However,  I cannot get any 'jump to source' or autocompletion when dealing with packages installed by opam:
![image](https://user-images.githubusercontent.com/18023393/35255676-21a0ed68-002b-11e8-9e46-c9a60177fe9f.png)
The ```Re``` module was installed by opam and already configured in merlin file(auto generated by jbuilder)
![image](https://user-images.githubusercontent.com/18023393/35255873-1e562640-002c-11e8-99d9-a1d46261a393.png)
But the intellisense doesn't work when hovering on ```Re.str```.

I installed reason toolchains via npm, and it seems that I can only jump to the std library under the global npm lib folder. Did I miss any extra configuration that could make this plugin work with opam-installed packages? Thx~

**Comment by 1yefuwang1** [comment]
It seems having ```PKG re``` in my merlin file is not enough for this plugin to locate packge ```re```, which is why I can't get any autocompletion. This problem can be fixed by manually adding the path of ```re``` into the merlin file.
I used vscode-ocaml(now deprecated) before, and , if my memory serves, ```PKG``` directive was enough to get me autocompletion.

**Comment by freebroccolo** [comment]
I am not sure if I am understanding correctly but it sounds like you are trying to combine tooling and library installed via `npm` with that of `opam`. Unfortunately, this is not a supported configuration and I would be very surprised if it worked with the other extension somehow. The problem is the paths will be set up differently in an incompatible way in the environment if you are running tooling installed one way or the other.

Basically you will need to choose whether to use `npm` or `opam` but not both. There isn't a way to make this work on the extension side unfortunately.
